osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/BundleService.java

```diff
@@ -132,6 +136,9 @@ private void checkIfAdditionalActionIsNeeded(final DeviceMessageMetadata deviceM
             } else if (action instanceof SetDeviceLifecycleStatusByChannelResponseDto) {
                 this.managementService
                         .setDeviceLifecycleStatusByChannel((SetDeviceLifecycleStatusByChannelResponseDto) action);
+            } else if (action instanceof EventMessageDataResponseDto) {
+                // Add EventType to EventDto
```

Comment:
the comment seems redundant, because the name of the method almost resembles the comment.

Reply:
Agree, removed comment